specs/badge.md

```diff
@@ -0,0 +1,80 @@
+# fast-badge
+
+## Overview
+
+*Badge* component is used to highlight an item and attract attention or flag status, such as “New”, “Sale”, or other short phrases.
+
+### Use Cases
+Keep the *badge* label short and succinct &ndash; it should not be a sentence or a long phrase (recommend 2-3 words). Short phrases work well, such as “App of the day,” but keep in mind that they will generally become longer when localized. Keep the text to nouns and adjectives if possible (e.g. “New” or “New today”), so the user won’t think it is an actionable element.
```

Comment:
This seems more like general design guidance as opposed to use cases; is there a better place for this to live?

Reply:
I'll remove for now. If add in some design guidance at a later date this could potentially live there.